app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -35,7 +41,23 @@ class WireNotificationManager @Inject constructor(
     }
 
     private suspend fun fetchAndShowCallNotificationsOnce(userId: QualifiedID) {
-        //TODO
+        //TODO for now GetIncomingCallsUseCase() returns valid data not from the first try.
+        // so it's possible to have scenario, when FCM comes informing us that there is a Call,
+        // but we don't get it from the first GetIncomingCallsUseCase() call.
+        // To cover that case we have this `intervalFlow().take(6)`
+        // to try get incoming calls 6 times, if it returns nothing we assume there is no incoming call
```

Comment:
Isn't the value of `CHECK_INCOMING_CALLS_TRIES` currently 10 instead of the 6 tries here mentioned? Which one is the intended one then?

Reply:
Why we have try 6 times and not any other number ? I suggest to keep observing and that's all ?
`.getIncomingCalls().collect { 
}
`